website/src/utils/search.ts

```diff
@@ -56,7 +56,7 @@ const getFieldOrColumnVisibilitiesFromQuery = (
 ): Map<string, boolean> => {
     const visibilities = new Map<string, boolean>();
     schema.metadata.forEach((field) => {
-        if (field.hideOnSequenceDetailsPage === true || !visibilitySelectableAccessor(field)) {
```

Comment:
This change was necessary to be able to show the "is revocation" and "version status" field in the visibility selector - but now a bunch of other things are also visible. I think this isn't an issue, but I'll just have to add `notSearchable: true` to a bunch of other fields.

I also noticed for example this:

```yaml
  - name: submittedDate
    type: string
    hideOnSequenceDetailsPage: true
    generateIndex: true
    autocomplete: true
    displayName: Date submitted (exact)
```

This is currently live that way. `autocomplete` and `generateIndex` imply that someone wanted to have these searchable - but because of `hideOnSequenceDetailsPage` it's actually not searchable ... curious.

Reply:
some other fields actually _do_ have `notSearchable` and `hideOnSequenceDetailsPage` set explicitly. I am now assuming that the changes are made are actually fine without setting more `notSearchable` fields. Maybe someone with more knowledge about the data specifics can have a look and verify which fields should be searchable but not visible etc.

Reply:
> someone wanted to have these searchable 

I wouldn't overthink this - especially in the Loculus config - there may be lots of accidental config

Reply:
Alright! Then I think It's fine the way it is. Thanks!